src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,52 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			log.Fatalf("Script not found: %s", path)
+			os.Exit(1)
+		}
+		// get the mode bits and check that the script is executable by someone
+		mode := info.Mode()
+		if mode & 0111 == 0 {
+			log.Fatalf("Script not executable: %s", path)
+			os.Exit(1)
+		}
+		log.Infof("Running script: %s", path)
+		if ctx.DryRun == true {
```

Comment:
style nitpick: it's common to just use booleans without an explicit comparison, and we should follow that